scripts/run-dependencies

```diff
@@ -9,12 +9,6 @@ HERE=$(realpath "$(dirname $0)")
 "$HERE"/../db/scripts/build
 "$HERE"/../db/scripts/run
 
-# Populate db with dummy data
```

Comment:
no longer needed :tada:  when the app runs, the scheduler immediately imports data from `outpack_server`.